packages/client/package.json

```diff
@@ -25,7 +25,8 @@
     "extract:translations": "bash extract-translations.sh",
     "generate-gateway-types": "graphql-codegen --config codegen.yml && prettier --write src/utils/gateway.ts",
     "build:clean": "rm -rf build",
-    "serve": "node serve-static.js"
+    "serve": "node serve-static.js",
+    "prune": "npx ts-prune -i '(root|gateway.ts|forms)'"
```

Comment:
maybe we can remove this 🤔 ts-prune isn't going to be supported going forward, the developer has stated that TS needs an official plugin for this